packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -162,7 +223,7 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
         uint256 _l2OutputIndex,
         Types.OutputRootProof calldata _outputRootProof,
         bytes[] calldata _withdrawalProof
-    ) external {
+    ) external whenNotPaused {
```

Comment:
This feels like the conservative approach, but there is an argument to be made that we should allow proofs to continue without delaying finalization.
In case we really want to delay their finalization, we can (at the moment) also use `deleteL2Outputs()` to postpone them.

Reply:
Going to update this to not pause deposits anymore.

I do see your point re: better UX. It feels more simple to me this way and its less tests that need to be written now

Reply:
OK